frontend/src/components/GameSettings/GameSettings.js

```diff
@@ -130,6 +133,13 @@ function GameSettings({ options, onChange }) {
     }
   }, []);
 
+  useEffect(() => {
+    const playerCount = state.playerIDs.length + state.newPlayerStaging.length;
+    setMinimumPlayerLimit(() =>
+      playerCount > config.maxPlayers.min ? playerCount : config.maxPlayers.min,
+    );
+  }, [state.playerIDs, state.newPlayerStaging]);
```

Comment:
Since this component is only used in the HostPregameScreen where no players will have been moved from staging into `players` is it needed for this to include `state.playerIDs` and update when it changes? 

Not that having it hurts anything, but if we aren't intending to use this anywhere else it obscures things a bit IMO.